eng/Versions.props

```
@@ -2,11 +2,11 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.11.8</VersionPrefix>
+    <VersionPrefix>17.11.9</VersionPrefix>
     <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

```suggestion
    <VersionPrefix>17.11.9</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

We try to keep this on one line for maximum merge-conflict-ness.